uaclient/api/api.py

```diff
@@ -18,6 +18,7 @@
     "u.pro.attach.magic.initiate.v1",
     "u.pro.attach.magic.wait.v1",
     "u.pro.attach.magic.revoke.v1",
+    "u.pro.detect.is_pro.v1",
```

Comment:
Naming bikeshed: the instance isn't pro yet, but it we're detecting if it _should_ be pro.

How about: "u.pro.detect.is_pro_eligible.v1"

of even: "u.pro.detect.should_auto_attach.v1"

thoughts?

Reply:
I don't know, the first one is a bit misleading because it might sound that machine is not pro yet, but it could be. However, our is to detect the running machine is already Pro.

Because of that, I think the second one is better. I will use it instead